sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot.c

```diff
@@ -0,0 +1,23 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot.h"
+
+#include "sw/device/silicon_creator/lib/error.h"
+
+void boot_svc_next_boot_bl0_slot_req_init(
+    boot_svc_next_boot_bl0_slot_req_t *msg, uint32_t next_slot) {
```

Comment:
input parameters come before output parameters:
```suggestion
    uint32_t next_slot, boot_svc_next_boot_bl0_slot_req_t *msg) {
```